templates/index.html

```diff
@@ -1,5 +1,21 @@
-{% extends 'base.html' %}
-{% block title %} {{title}}{% endblock %}
+{% extends 'base3.html' %}
 {% block content %}
-<h1>Добро пожаловать</h1>
-{% endblock %}
+<form action="/", method="post">
+    {{ form.csrf_token }}
+    {% for field in form if field.name != 'csrf_token' %}
+        <p>
+        {{ field.label }}<br>
+        {{ field }}
+        {% if field.errors %}
+        <ul class="alert alert-danger">
+        {% for error in field.errors %}
+        <li>{{ error }}</li>
+        {% endfor %}
+        </ul>
+        {% endif %}
+        </p>
+        {% endfor %}
+
+    <input type="submit", value="регистрация">
```

Comment:
То эту кнопку можно не создавать, так как она будет в объекте формы.
```suggestion
            <p>
	        {{ form.submit() }}
	    </p>
```